test/quantization/test_quantize_script.py

```diff
@@ -1371,7 +1352,8 @@ def forward(self, x):
                              (NonQuantizedAddScalar(), False),
                              (NonQuantizedInplaceAddScalar(), False)]:
             op = "quantized::add_scalar" if quantized else "aten::add"
-            m = self._test_op_impl(m, data, op)
+            # TODO: fix debug=True numerics
```

Comment:
For compactness, can we collapse the 4 models per op into a single model? Can we have a single forward doing both non-inplace and inplace operations? Since you duplicate this structure for other ops, this can make the code easy to read.

Reply:
I can try it in a separate PR, I'm not sure if that is scriptable